plugins/cloud-aws/src/main/java/org/elasticsearch/cloud/aws/blobstore/S3BlobStore.java

```diff
@@ -71,13 +71,28 @@ public S3BlobStore(Settings settings, AmazonS3 client, String bucket, @Nullable
         }
 
         this.numberOfRetries = maxRetries;
-        if (!client.doesBucketExist(bucket)) {
-            if (region != null) {
-                client.createBucket(bucket, region);
-            } else {
-                client.createBucket(bucket);
+
+        int retry = 0;
+        while (retry <= this.numberOfRetries) {
```

Comment:
Hmm, probably I should do that.
But even we set maxRetries to 0, we still get one try here.  Basically I did the same what we have [there] (https://github.com/elastic/elasticsearch/blob/master/plugins/cloud-aws/src/main/java/org/elasticsearch/cloud/aws/blobstore/DefaultS3OutputStream.java#L222)